package protoFile.shop.buyItemInMyshop;
message buyItemInMyshopRequest{
     required int32 id = 1;
     required int32 itemTemplateId = 2;
}

message buyItemInMyshopResponse{
     required bool result = 1;
     optional string message = 2;
}